share/scripts/inverse/run_gromacs.sh

```diff
@@ -133,14 +153,15 @@ else
   echo "${0##*/}: No walltime defined, so no time limitation given to $mdrun"
 fi
 
-#>gmx-5.1 has new handling of bonded tables
-if [[ ${mdrun_opts} != *tableb* ]]; then
+#gmx-5.1 has new handling of bonded tables - is it true only for ver.5.1 or above too???
```

Comment:
The `-tableb` option is broken in gromacs-5.1.[012], see #210 for details and there is no workaround for that. Bonded tables just don't work in these versions!

And this block is just to ensure that if the user already has `-tableb` listed in `${mdrun_opts}`, we don't do anything.

On a related note, we should an error message about that bug.